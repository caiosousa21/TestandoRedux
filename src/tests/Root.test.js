import React from 'react';
import Root from '../containers/Root';
import Enzyme, { shallow, mount } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

Enzyme.configure({ adapter: new Adapter() })

describe('index', () => {
    it('Renderizando Root', () => {
        const wrapper = shallow(<Root />);
        expect(wrapper.exists()).toBe(true);
    })
    it('Testando interface com snapshots', () => {
        const tree = renderer.create(<Root />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})