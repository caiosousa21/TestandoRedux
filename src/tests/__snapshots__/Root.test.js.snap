// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index Testando interface com snapshots 1`] = `
<div>
  <span>
    <h1>
      reactjs
    </h1>
    <select
      onChange={[Function]}
      value="reactjs"
    >
      <option
        value="reactjs"
      >
        reactjs
      </option>
      <option
        value="frontend"
      >
        frontend
      </option>
    </select>
  </span>
  <p />
  <h2>
    Loading...
  </h2>
</div>
`;
